client/src/lib.rs

```diff
@@ -186,6 +201,13 @@ pub struct Client {
 #[derive(Clone)]
 pub struct Frontend(Client);
 
+#[derive(serde::Deserialize)]
+struct TransferParams {
+    donor: Option<linera_base::identifiers::Owner>,
```

Comment:
`source` ?

Reply:
Looks like you're behind `testnet-babbage` btw

Reply:
I am, I'll make a new PR to port both examples to Babbage :)

Reply:
The pair for `recipient` is `donor`, and for `source` is `target`.  Happy to do either but I'd like a matching pair.